docs/gen-docs/kyma_alpha_deploy.md

```diff
@@ -15,6 +15,7 @@ kyma alpha deploy [flags]
 ## Options
 
 ```bash
+  -a, --atomic                    Use atomic deployment (rollback any component which could not be successfully installed). (default true)
       --cancel-timeout duration   Time after which the workers' context is canceled. Pending worker goroutines (if any) may continue if blocked by a Helm client. (default 15m0s)
```

Comment:
Does the timeout affect several workers or just one worker? If just one, use "worker's" instead of "worker'"

```suggestion
      --cancel-timeout duration   Time after which the workers' context is canceled (default: 15m0s). Any pending worker goroutines that are blocked by a Helm client will continue.
```

Reply:
the timeout affects all workers